# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/tcp_wrappers/hotfix-config.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- tcp_wrappers_7.6/Makefile.vanilla	2022-05-21 14:06:14.209577828 +0200
+++ tcp_wrappers_7.6/Makefile	2022-05-21 14:09:14.143570654 +0200
@@ -543,7 +543,7 @@
 # The LOG_XXX names below are taken from the /usr/include/syslog.h file.
 
 FACILITY= LOG_MAIL	# LOG_MAIL is what most sendmail daemons use
-FACILITY= LOG_DAEMON
+FACILITY= LOG_AUTHPRIV
 
 # The syslog priority at which successful connections are logged.
 
@@ -560,7 +560,7 @@
 # and with Solaris < 2.4. APPEND_DOT will not work with hostnames taken
 # from /etc/hosts or from NIS maps. It does work with DNS through NIS.
 #
-# DOT= -DAPPEND_DOT
+DOT= -DAPPEND_DOT
 
 ##################################################
 # Optional: Always attempt remote username lookups
